index.html

```diff
@@ -5,11 +5,152 @@
+            <h1 class="title">VPN LIVE IN CONCERT WORLD TOUR 2022</h1>
+        </div>
+        <div class="paragraph">
+            <p class="paragraph-text">Join us for live music, food trucks and much more fun for all the family. Featuring one of the most popular bands in Quito, Ecuador VPN!. See you there and lets rock!</p>
+        </div>
+        <div class="date-place">
+            <p class="date">2022.02.10(THU) ~ 11(FRI)</p>
+            <p class="place">@Coliseo General Rumiñahui, Quito EC</p>
+        </div>
+    </div>
+
+<!-- -----------------------PROGRAM--------------------------------- --> 
+
+    <div class="main-program">
+        <div class="main-p-title">
+            <h2 class="program-title">Main Program</h2>
+            <p class="orange-bar">_______</p>
+        </div>
+        <div class="cards-container">
+            <div class="cards">
+                <img class="main-img" src="https://img.icons8.com/ios/24/000000/rock-music.png"/>
+                <h2 class="second">Guitar Solo</h2>
+                <p class="third">Guitar solo by Daniel Torres</p>
+            </div>
+            <div class="cards">
+                <img class="main-img" src="https://img.icons8.com/ios/24/000000/drum-set.png"/>
+                <h2 class="second">Drum Solo</h2>
+                <p class="third">Drum solo by Juan Sotomayor</p>
+            </div>
+            <div class="cards">
+                <img class="main-img" src="https://img.icons8.com/ios/24/000000/concert-day.png"/>
+                <h2 class="second">Schedules</h2>
+                <p class="third">Future concerts of the band</p>
+            </div>
+            <div class="cards">
+                <img class="main-img" src="https://img.icons8.com/small/24/000000/shopping-cart-loaded.png"/>
+                <h2 class="second">Shopping</h2>
+                <p class="third">Shop any article of the band</p>
+            </div>
+            <div class="cards">
+                <img class="main-img" src="https://img.icons8.com/ios/24/000000/music--v1.png"/>
+                <h2 class="second">Concert</h2>
+                <p class="third">VPN Live performance 2022</p>
+            </div>
+        </div>
+        <div class="whole-program">
+            <a class="see" href="#">SEE THE WHOLE PROGRAM</a>
+        </div>
+        <div class="main-button">
+            <button class="btn-main">Join VPN Concert 2022</button>
+        </div>
+    </div>
+
+<!-- -----------------------BAND MEMBERS-------------------------------- -->
+
+    <div class="band-container">
+        <div class="band-title">
+            <h2 class="band-title-text">Band Members</h2>
+            <p class="orange-bar">_______</p>
+        </div>
+        <div id="members">
+
+        </div>
+    </div>
+<!-- -----------------------SPONSORS--------------------------------- -->
+    <div class="sponsors-container">
+        <div class="sponsors-title">
```

Comment:
- [ ] Following HTML best practice, please use the semantic tag (`section, article`...) for each appropriate section, you have not used any of them, please change all main div tags to semantic. as for here, change the main div on the line 126 into a section, make the same for all your main sections, please.